### DISCUSSION TITLE
Feature Suggestions

### DISCUSSION THREAD

**Discussion by hilltopmediadesign** [uxbox/uxbox#828] [Ideas]
I'm thoroughly impressed with how much has been accomplished in this project already. I'm looking forward to moving our team over to Penpot as you move towards 1.0.

Here's a few features suggestions from a regular figma user:

- Adjust values by ctrl/cmd clicking in a value field (i.e. opacity) and dragging left/right to decrease/increase value. I believe Davinci Fusion does this. This could really improve UI effectiveness.
- Export PSD or PDF as layers.
- Scale text with mouse (essential)
- Stretch text. Not just scale. Figma currently does not do this unless you convert layer to a vector.
- Right click on text layer > convert to vector. (I believe it's flatten in Figma)

Hope that's helpful.

Cheers.

**Comment by elhombretecla** [comment]
Hi @hilltopmediadesign thank you for your feedback!!!

We take note of your proposals and we will evaluate them in depth. They are very interesting and useful.

:)

**Comment by rowild** [comment]
Would love to upvote specifically the "Text to Vector" (Figma: "flatten") feature!
Thank you, guys at Penpot! You are truly amazing!
